packages/server/graphql/mutations/changeTaskTeam.ts

```diff
@@ -54,7 +58,64 @@ export default {
       })
     }
 
+    // The task might have been pushed by someone else for viewer (`userId !== accessUserId`).
+    // In that case we still try to use the viewer's target team authentication, but fall back to the
+    // accessUser's in case it is present for the target team.
+    const [sourceTeamAuth, targetTeamAuth, accessUsersTargetTeamAuth] =
+      task.integration?.service === 'jira'
+        ? await Promise.all([
+            dataLoader.get('freshAtlassianAuth').load({teamId: task.teamId, userId: viewerId}),
```

Comment:
these might be good candidates for `loadMany`, although those have the possibility of returning an Error instance.
I wrote an example of what that might look like, let me know what you think

Reply:
I don't really like the `loadMany` signature and it's just a wrapper for `Promise.all(load(...))`. Question is whether we want to silently ignore errors and convert them to null or let them bubble up. I tend to default to just let these bubble up since we don't expect any errors here. But I can reduce repetition.

Reply:
agreed, the signature used to be so much nicer! https://github.com/graphql/dataloader/pull/216
we're already silently ignoring any errors inside the batchFn, i wish we could tell the dataloader that we don't need any errors that come out of it.

mapping & wrapping in Promise.all is probably the cleanest way as long as we can guarantee that none of them will throw.